docs/_snippets/portable-stories-vitest-set-project-annotations.md

```diff
@@ -0,0 +1,53 @@
+```tsx filename="setupTest.ts" renderer="react" language="ts"
+import { beforeAll } from 'vitest';
+import { render as testingLibraryRender } from '@testing-library/react';
+import { setProjectAnnotations } from '@storybook/react';
+import * as addonAnnotations from 'my-addon/preview';
+import * as previewAnnotations from './.storybook/preview';
+
+const annotations = setProjectAnnotations([
+  previewAnnotations,
+  addonAnnotations,
+  // You MUST provide this option to use portable stories with vitest
```

Comment:
Similar to my comment about the related callout. I simplified this comment, but I'm not sure if it's actually more helpful.